src/Sarif.Viewer.VisualStudio/Resources.resx

```diff
@@ -117,6 +117,12 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="ConfirmDownload_DialogMessage" xml:space="preserve">
+    <value>The source file {0} will now be downloaded. To abort the download, click Cancel.
```

Comment:
Unfortunately this was the best I could do with what MessageBox provides. Can't customize the buttons much. Before I merge I'll take another look around and see if there is another dialog I can use. Otherwise I'd have to roll my own which I don't think looks very legit.